pkg/util/fast_int_set.go

```diff
@@ -17,10 +17,11 @@ package util
 import (
 	"bytes"
 	"fmt"
+	"sort"
 )
 
-// FastIntSet keeps track of a set of integers. It is very fast when the values
-// are small. It is not thread-safe.
+// FastIntSet keeps track of a set of integers. It does not perform any
+// allocations when the values are small. It is not thread-safe.
 type FastIntSet struct {
 	smallVals uint64
 	largeVals map[uint32]struct{}
```

Comment:
Yes, but it is not trivial to implement correctly and it would be throwaway work because we probably do want to switch to the `Sparse` implementation, and it would utterly useless for this change because we normally don't have 64 columns, and if we do, I'm willing to bet the "hot path" of such a query won't be representing column groups in this structure :)